otcextensions/sdk/vpc/v1/subnet.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from openstack import resource
+
+
+class ExtraDHCPOpt(resource.Resource):
+    #: Specifies the NTP server address configured for the subnet.
+    opt_value = resource.Body('opt_value')
+    #: Specifies the NTP server address name configured for the subnet.
+    #: Currently, the value can only be set to ntp.
+    opt_name = resource.Body('opt_name', default='ntp')
```

Comment:
maybe both of them should have defaults

Reply:
The `opt_value` is not mandatory and has the following limitations:
```
The option ntp for opt_name indicates the NTP server configured for the subnet. Currently, only IPv4 addresses are supported. A maximum of four IP addresses can be configured, and each address must be unique. Multiple IP addresses must be separated using commas (,). The option null for opt_name indicates that no NTP server is configured for the subnet. The parameter value cannot be an empty string.
```
And which IP address value can be used in this case?